src/R/Components/Impl/PackageManager/IRPackageManager.cs

```diff
@@ -2,11 +2,18 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.R.Components.PackageManager.Model;
 
 namespace Microsoft.R.Components.PackageManager {
     public interface IRPackageManager : IDisposable {
         IRPackageManagerVisualComponent VisualComponent { get; }
 
         IRPackageManagerVisualComponent GetOrCreateVisualComponent(IRPackageManagerVisualComponentContainerFactory visualComponentContainerFactory, int instanceId = 0);
+
+        Task<IList<RPackage>> GetInstalledPackagesAsync();
```

Comment:
If there's no specific meaning to the order of packages, perhaps `ICollection` is more appropriate here.